### PR TITLE
FALCON-1845 Retries fix when entity was deleted during instance rerun

### PR DESCRIPTION
**Pull request by pavankumar526**

### CODE REVIEW

File: rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java [hunk 1]
```diff
@@ -112,6 +112,8 @@ public void onSuccess(WorkflowExecutionContext context) throws FalconException {
     public void onFailure(WorkflowExecutionContext context) throws FalconException {
         // Re-run does not make sense when killed by user.
         if (context.isWorkflowKilledManually()) {
+            LOG.debug("User workflow: {} killed manually of instance: {} belongs to entity: {}, so retry won't happen ",
```
Comment (pallavi-rao) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java]: Minor nit on the phrasing of the message. Can we rephrase to "Workflow {}; Instance {}; Entity {}, killed manually by user. Will not retry."

**Comment by pallavi-rao** [comment]
+1 after from the minor nit.

File: rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java [hunk 2]
```diff
@@ -112,6 +112,8 @@ public void onSuccess(WorkflowExecutionContext context) throws FalconException {
     public void onFailure(WorkflowExecutionContext context) throws FalconException {
         // Re-run does not make sense when killed by user.
         if (context.isWorkflowKilledManually()) {
+            LOG.debug("Workflow: {} Instance: {} Entity: {}, killed manually by user. Will not retry.",
```
Comment (pavankumar526) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java]: Changed based on comments

File: rerun/src/main/java/org/apache/falcon/rerun/handler/AbstractRerunConsumer.java [hunk 3]
```diff
@@ -88,7 +88,6 @@ public void run() {
                     LOG.warn("Entity {} of type {} doesn't exist in config store. Rerun "
                                     + "cannot be done for workflow ", message.getEntityName(),
                             message.getEntityType(), message.getWfId());
-                    return;
```
Comment (sandeepSamudrala) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/AbstractRerunConsumer.java]: It should be continue else the message below would get logged. (Error in rerun consumer) ?

**Comment by sandeepSamudrala** [comment]
:+1:  other than that one comment

File: rerun/src/main/java/org/apache/falcon/rerun/handler/AbstractRerunConsumer.java (hunk 3 above)
Comment (pavankumar526) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/AbstractRerunConsumer.java]: Makes sense will change

**Pull request closed without merging**